### PR TITLE
BRUNO's wallet diagnostics

### PR DESCRIPTION
**Pull request by skylarbarrera**
stealing bruno's commits for clout

PoW: https://cloud.skylarbarrera.com/Screen-Recording-2021-06-15-11-36-48.mp4

Fixes RNBW-1257

### CODE REVIEW

**Review by brunobar79 — APPROVED** [review]
🙏 🙏 🙏

File: src/screens/WalletDiagnosticsSheet.js [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+          <Fragment>
+            <Column>
+              {pkeys?.map(key => (
+                <ItemRow data={key} key={`row_${key.username}`} />
+              ))}
+            </Column>
+          </Fragment>
+        )}
+
+        {keys?.length > 0 && (
```
[the remark this answers is not in the document]
Comment (skylarbarrera) [reply src/screens/WalletDiagnosticsSheet.js:299]: oops

**Comment by jinchung** [comment]
* Confirmation alert (success or failure) after hitting Restore
* In the recording I don't see the alert pop up after "Copy Secret" on the android version
* Alert explaining how it will replace that wallet before it restores

File: src/screens/WalletDiagnosticsSheet.js [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+            })
+        );
+        setKeys(processedKeys);
+      } catch (e) {
+        logger.sentry('Error processing keys for wallet diagnostics', e);
+        const customError = new Error('WalletDiagnostics init failed');
+        captureException(customError);
+      }
+    };
+    setTimeout(() => {
```
Comment (jinchung) [inline src/screens/WalletDiagnosticsSheet.js:226]: @brunobar79 do you mind adding a comment before the setTimeout that we're doing this bc the flow is coming from a user tapping AppVersionStamp a bunch of times and we don't want them accidentally restoring

Comment (skylarbarrera) [reply src/screens/WalletDiagnosticsSheet.js:226]: wondering if we can just remove the tapping aspect? its kinda finicky already and i doubt anyone taps that on accident

Comment (brunobar79) [reply src/screens/WalletDiagnosticsSheet.js:226]: nah, it works fine now

**Comment by brunobar79** [comment]
Closing bc case-sensitive shit that I'm not willing to fight against today

**Pull request closed without merging**